tfhe/src/integer/u256.rs

```diff
@@ -269,6 +269,23 @@ impl std::ops::AddAssign<Self> for U256 {
     }
 }
 
+impl std::ops::BitOr<Self> for U256 {
+    type Output = Self;
+
+    fn bitor(mut self, rhs: Self) -> Self::Output {
+        self |= rhs;
+        self
+    }
+}
+
+impl std::ops::BitOrAssign<Self> for U256 {
+    fn bitor_assign(&mut self, rhs: Self) {
+        for (self_word, rhs_word) in self.0.iter_mut().zip(rhs.0) {
+            *self_word |= rhs_word;
+        }
+    }
+}
+
```

Comment:
unit test when you get a chance or backlog issue

Reply:
I am creating a backlog issue where i'll keep track of every ops we need (and test) to be able to have U256 an implementor of the UnsignedInteger Trait (which sadly will be blocked by the fact that we don't yet have a I256)